UI/src/app/components/popups/open-save-dialog/open-save-dialog.component.html

```diff
@@ -0,0 +1,27 @@
+<h3>{{data.header}}</h3>
+<mat-icon class="close"
+          mat-dialog-close>close
+</mat-icon>
+<div mat-dialog-content>
+  <mat-form-field appearance="standard">
+    <mat-label>{{data.dropdownLabel}}</mat-label>
+  <mat-select *ngIf="!data.input" #item [(value)]="resultValue">
```

Comment:
I think we should pass type instead of input into data object e.g.
```html
*ngIf="data.type === 'select'"
```

It'll extend capabilities to pass other types in future: (number/checkbox etc.) .

Reply:
Done